test/test_preprocessor.py

```diff
@@ -39,6 +39,12 @@ def test_preprocessor(implementation: pqclean.Implementation):
                     ("crypto_sign/dilithium2/aarch64/fips202x2.c" not in file) and
                     ("crypto_sign/dilithium3/aarch64/fips202x2.c" not in file) and
                     ("crypto_sign/dilithium5/aarch64/fips202x2.c" not in file) and
+                    ("crypto_kem\\kyber512\\aarch64\\fips202x2.c" not in file) and
+                    ("crypto_kem\\kyber768\\aarch64\\fips202x2.c" not in file) and
+                    ("crypto_kem\\kyber1024\\aarch64\\fips202x2.c" not in file) and
+                    ("crypto_sign\\dilithium2\\aarch64\\fips202x2.c" not in file) and
+                    ("crypto_sign\\dilithium3\\aarch64\\fips202x2.c" not in file) and
+                    ("crypto_sign\\dilithium5\\aarch64\\fips202x2.c" not in file) and
```

Comment:
it might make sense to use Pathlib and simplify this to:

```py
Path(file).match("dilithium[235]/aarch64/fips202x2.c")
```

Reply:
Done. I hope this works on windows like that.

Reply:
Yeah I tested it with `PureWindowsPath("foo\\bar").match`